documentation/asciidoc/accessories/camera/libcamera_options_still.adoc

```diff
@@ -0,0 +1,128 @@
+
+Clearly the application needs to supply EXIF tags that contain specific camera data (like the exposure time). But for other tags that have nothing to do with the camera, a reasonable workaround would simply be to add them _post facto_, using something like `exiftool`.
+
+Example: `libcamera-still -o test.jpg --exif IDO0.Artist=Someone`
+
+----
+	--timelapse			Time interval between timelapse captures <milliseconds>
+----
+
+This puts `libcamera-still` into timelapse mode where it runs according to the timeout (`--timeout` or `-t`) that has been set, and for that period will capture repeated images at the interval specified here. (`libcamera-still` only.)
+
+Example: `libcamera-still -t 100000 -o test%d.jpg --timelapse 10000` captures an image every 10s for about 100s.
+
+----
+	--framestart			The starting value for the frame counter <number>
+----
+
+When writing counter values into the output file name, this specifies the starting value for the counter.
+
+Example: `libcamera-still -t 100000 -o test%d.jpg --timelapse 10000 --framestart 1` captures an image every 10s for about 100s, starting at 1 rather than 0. (`libcamera-still` only.)
+
+----
+	--datetime			Use date format for the output file names
+----
+
+Use the current date and time to construct the output file name, in the form MMDDhhmmss.jpg, where MM = 2-digit month number, DD = 2-digit day number, hh = 2-digit 24-hour hour number, mm = 2-digit minute number, ss = 2-digit second number. (`libcamera-still` only.)
+
+Example: `libcamera-still --datetime`
+
+----
+	--timestamp			Use system timestamps for the output file names
+----
+
+Uses the current system timestamp (the number of seconds since the start of 1970) as the the output file name. (`libcamera-still` only.)
+
+Example: `libcamera-still --timestamp`
+
+----
+	--restart			Set the JPEG restart interval <number>
+----
+
+Sets the JPEG restart interval to the given value. Default is zero.
+
+Example: `libcamera-still -o test.jpg --restart 20`
+
+----
+	--keypress,	-k		Capture image when Enter pressed
+----
+
+This switches `libcamera-still` into keypress mode. It will capture a still image either when the timeout expires or the Enter key is pressed in the terminal window. Typing `x` and Enter causes `libcamera-still` to quit without capturing.
+
+Example: `libcamera-still -t 0 -o test.jpg -k`
+
+----
+	--signal,	-s		Capture image when SIGUSR1 received
+----
+
+This switches `libcamera-still` into signal mode. It will capture a still image either when the timeout expires or a SIGUSR1 is received. SIGUSR2 will cause `libcamera-still` to quit without capturing.
+
+Example:
+
+`libcamera-still -t 0 -o test.jpg -s &`
+
+then
+
+`kill -SIGUSR1 $!`
+
+----
+	--thumb,			Set thumbnail parameters <w:h:q>
```

Comment:
There shouldn't be a comma after `--thumb` because it doesn't have a short-from option?

Reply:
Will fix that.